app/javascript/components/search/genes/GeneKeyword.js

```diff
@@ -48,11 +48,13 @@ export default function GeneKeyword({ placeholder, helpTextContent }) {
 
   /** Converts any current typed free text to a gene array entry */
   function syncGeneArrayToInputText() {
-    if (!inputText) {
+    const inputTextTrimmed = inputText.trim().replace(',', '')
```

Comment:
Shouldn't `replace()`  take a regex rather than `','`?  That will only catch the first instance.  It is unclear to me if this would be the entire search string or just the last token.

Reply:
good point.  it doesn't matter here since the user is prevented from ever entering more than one comma by the keydown handler clearing the inputText after a comma is entered.  But it can't hurt to be safe--i'll update it